src/main/java/project/linkarchive/backend/user/repository/RefreshTokenRepository.java

```diff
@@ -0,0 +1,8 @@
+package project.linkarchive.backend.user.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import project.linkarchive.backend.user.domain.RefreshToken;
+
+public interface RefreshTokenRepository extends JpaRepository<RefreshToken, Long> {
+    RefreshToken findByUserId(Long userId);
```

Comment:
```suggestion
    RefreshToken findByUserId(Long userId);
```

```
   Optional<RefreshToken> findByUserId(Long userId);
```
Optional 로 바꿔주는건 어떨까요??